### ISSUE TITLE
languageserver 0.3.3 causes constant Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by hermidalc** [REditorSupport/languageserver#161]
In Atom 1.42 or 1.43 and atom-language-r 1.4.8 on Linux r-languageserver 0.3.3 causes constant ` Uncaught TypeError: Cannot read property 'length' of undefined` errors once you start typing code in an R file.  The problem is severe enough that it makes Atom unusable to edit R files.  When I revert back to 0.3.2 the problem goes away.

**Comment by randy3k** [comment]
I cannot reproduce it on Atom 1.40.1 and languageserver 0.33 in macOS. Let me try again with Atom 1.43.

**Comment by randy3k** [comment]
I upgraded Atom to 1.43.0 and still couldn't reproduce the error. Do you have the latest `ide-r` and `atom-ide-ui`?

**Comment by hermidalc** [comment]
> I upgraded Atom to 1.43.0 and still couldn't reproduce the error. Do you have the latest `ide-r` and `atom-ide-ui`?

Yes I have the latest, neither has been updated since 2018.  If you cannot reproduce the issue with the same setup then could it be Atom on Linux vs macOS?  I can reproduce the issue consistently when I update r-languageserver 0.3.2 -> 0.3.3

**Comment by randy3k** [comment]
Could you open the developer console and run `atom.config.set('core.debugLSP', true)`.? That might give us some idea what is going on.

**Comment by hermidalc** [comment]
Really sorry for the long delay with this.  I was still using the old `atom-ide-ui` which I realized that I needed to migrate to Atom IDE Community packages instead.  Using Atom IDE Community packages instead of `atom-ide-ui` then I have no issues with latest `r-languageserver=0.3.4`.

BTW for some reason on conda-forge there is an `0.3.5` but only for R 3.5 not 3.6?
